.github/workflows/regressions.yml

```diff
@@ -114,7 +114,9 @@ jobs:
                   arguments: "clean :test --tests \"org.rustPerformanceTests.CustomRealProjectAnalysisTest.test\""
 
             - name: Calculate regressions
-              run: python scripts/calculate_regressions.py --name ${{ matrix.project.name }}
+              run: |
+                  git checkout ${{ github.sha }}
```

Comment:
Let's extract this line into separate step to be consistent with `Checkout base version` step, for example